gradle-server/src/main/java/com/github/badsyntax/gradle/handlers/GetBuildHandler.java

```diff
@@ -104,7 +104,7 @@ public void run() {
 			File initScript = PluginUtils.createInitScript();
 			List<String> arguments = new ArrayList<>();
 			String debugPlugin = System.getenv("VSCODE_DEBUG_PLUGIN");
-			if (debugPlugin.equals("true")) {
+			if (debugPlugin != null && debugPlugin.equals("true")) {
```

Comment:
good catch😆 

BTW, `"true".equals(debugPlugin)` or string utils would also do the trick. (never mind, just fyi)

Reply:
👍, let's use `"true".equals(debugPlugin)` to make this clear.

Reply:
null check is also clear, just personal tastes.